content/data-loading.md

```diff
@@ -607,7 +607,7 @@ Things can get more complicated; for instance you may want to deal with document
 
 <h3 id="publications-as-rest">Accessing a publication as a REST endpoint</h3>
 
-The opposite scenario occurs when you want to publish data to be consumed by a 3rd party, typically over REST. If the data we want to publish is the same as what we already publish via a publication, then we can use the [simple:rest](https://atmospherejs.com/simple/rest) package to do this really easily.
```

Comment:
👍 "More easily" would have also been acceptable, I think.  Removing it is great.  (and _some_  future version of the docs that had a CI test/linting step which forbid terms like _easy_ would welcome this change.)